src/main/java/com/zendesk/maxwell/producer/MaxwellRedisProducer.java

```diff
@@ -64,12 +84,20 @@ private String generateChannel(RowIdentity pk){
 		return channel;
 	}
 
+	private Jedis getJedisResource() {
+		if (jedisSentinelPool != null) {
+			return jedisSentinelPool.getResource();
+		}
+
+		return jedisPool.getResource();
+	}
```

Comment:
rather than these two `if` statements, can we just make `jedisPool` and instance of the common parent, `Pool<Jedis>`?